anvio/utils.py

```diff
@@ -381,6 +382,25 @@ def gzip_decompress_file(input_file_path, output_file_path=None, keep_original=T
 
     return output_file_path
 
+def tar_extract_file(input_file_path, output_file_path=None, keep_original=True):
+    filesnpaths.is_file_exists(input_file_path)
+
+    if not tarfile.is_tarfile(input_file_path):
```

Comment:
this should go into `filesnpaths` module so it is more broadly accessible to the codebase.

Reply:
I mean `tarfile.is_tarfile` and the error message if not should be accessible as `filesnpaths.is_tar_file`